src/assets/tooltiptext.js

```diff
@@ -1,3 +1,20 @@
 module.exports = {
-    price_impact: ""
+    price_impact: "",
+    //borrow page tooltips
```

Comment:
The following tooltips are not needed:
Received DAI, ETH Exposure, DAI available from ETH